utils/rpms/daoscov.spec

```diff
@@ -0,0 +1,705 @@
+%define daoshome %{_exec_prefix}/lib/%{name}
```

Comment:
This is absolutely not the way to do this.  Maintaining two different spec files is **definitely** NOT something we want to do.  It's enough to just maintain one.  You need to make the existing spec file conditionally do what you need done differently.